1. .Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16.Найти доверительный интервал для оценки математического ожидания с надежностью 0.95,если выборочная средняя M = 80, а объем выборки n = 256.

In [1]:
import numpy as np

In [2]:
M = 80
stdd = 16
a = 5
a_2 = a/2
z = 1.96
n = 256

In [3]:
M+(z*stdd/np.sqrt(n))

81.96

In [4]:
M-(z*stdd/np.sqrt(n))

78.04

с вероятностью 95% интервал [78.04; 81.96] захватит истинное генеральной совокупности

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,получены опытные данные:6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,оценить истинное значение величины X при помощи доверительного интервала, покрывающего этозначение с доверительной вероятностью 0,95.

In [5]:
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [6]:
x.sort()

In [7]:
sr1 = np.mean(x)
sr1

6.590000000000001

In [8]:
sr_std1 = np.std(x, ddof=1)
sr_std1

0.4508017549014448

In [15]:
M2 = 6.59
stdd2 = 0.45
n2 = 10

In [16]:
t = 2.262

In [17]:
M2+(t*stdd2/np.sqrt(n2))

6.911888243028539

In [18]:
M2-(t*stdd2/np.sqrt(n2))

6.268111756971461

с вероятностью 95% интервал [6.27; 6.9] захватит истинное генеральной совокупности

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметроказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

H0: диаметр = 17 мм
H1: диаметр = 17.5 мм

In [34]:
0.5/0.1

5.0

In [20]:
Zr = 0.5/0.2
Zr

2.5

In [21]:
Ztab = 1.645
Ztab

1.645

In [22]:
2.5>1.654

True

H0 не верна. H1 верна на уровне значимости 5%

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек.
Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно,
что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [24]:
x1 = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

H0: m0 = 200; H1: m != m0

In [25]:
x1.sort()

In [26]:
Mx1 = np.mean(x1)
Mx1

198.5

In [27]:
sr_stdx1 = np.std(x1, ddof=1)
sr_stdx1

4.453463071962462

In [28]:
tras = ((Mx1-200)*(10)**0.5)/4.453
tras

-1.0652181653385513

ttab: -3.25;3.25

-1.06 попадает в интервал. H0 верна на уровне значимости 1%. Утверждение продавца верно.